site/content/docs/4.3/thanks.md

```diff
@@ -1,34 +1,34 @@
 ---
 layout: docs
-title: Thanks
-description: To say thanks to our sponsors and services which support us.
+title: Support the team
```

Comment:
Note that we still need the description to be longer

Reply:
Yup, that's still a to do.